### ISSUE TITLE
Can't get updated topic_embeddings_ when using pre-calculated document embeddings?

### ISSUE THREAD

**Issue by nhansendev** [MaartenGr/BERTopic#2316]
### Have you searched existing issues?  🔎

- [x] I have searched and found no existing issues

### Desribe the bug

I can use the `update_topics` function by passing in the appropriate documents and my manually generated topic labels, and using the auto-generated embeddings. This works as intended and I get the updated `topic_embeddings_`.
`topic_model.update_topics(docs, topics=topics)`

However, if I calculate the document embeddings myself and pass them to BERTopic like so:
`topic_model.fit_transform(docs, embeddings=embeddings)`
then the `topic_embeddings_` are not updated after using `update_topics`. This is because the call to `_create_topic_vectors` within `update_topics` is done without any arguments being passed, and would normally use `self.embedding_model`, which is None when I define my own embeddings. 

How should I perform the update to get new `topic_embeddings_` in this case? 
Could a kwarg for passing pre-calculated embeddings to `update_topics` be added?

### BERTopic Version

Version: 0.16.4

**Comment by nhansendev** [comment]
A quick test of adding an `embeddings` kwarg to `update_topics` seems to work as intended:

```
if embeddings is not None:
    self._create_topic_vectors(documents, embeddings)
else:
    self._create_topic_vectors()
```

**Comment by MaartenGr** [comment]
Thanks for sharing this. This seems like a reasonable request and indeed straightforward to implement. Considering you already ran some tests, are you interested in creating a PR?

**Comment by nhansendev** [comment]
> Thanks for sharing this. This seems like a reasonable request and indeed straightforward to implement. Considering you already ran some tests, are you interested in creating a PR?

My testing has consisted of "yeah, that looks about the same as I expected" :)
